packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -23,24 +23,31 @@ export class UsaBanner extends React.PureComponent {
       this.props.locale === 'es' ? SpanishTranslations.usaBanner : EnglishTranslations.usaBanner;
 
     return (
-      <div className="ds-c-usa-banner">
+      <section className="ds-c-usa-banner" aria-label="Official government website">
         <header
           className={`ds-c-usa-banner__header ${
             this.state.isBannerOpen ? 'ds-c-usa-banner__header--expanded' : ''
           }`}
         >
-          <UsaFlagIcon className="ds-c-usa-banner__header-flag" />
-          <p className="ds-c-usa-banner__header-text">
-            <span>{t.bannerText}</span>
-            <button
-              onClick={this.handleToggleBanner}
-              className="ds-c-usa-banner__button"
-              aria-expanded={this.state.isBannerOpen}
-              aria-controls="gov-banner"
-            >
-              {t.bannerActionText}
-            </button>
-          </p>
+          <div className="ds-c-usa-banner__inner">
+            <p className="ds-c-usa-banner__header-text">
+              <UsaFlagIcon className="ds-c-usa-banner__header-flag" />
+            </p>
+            <p className="ds-c-usa-banner__header-text">
+              <span>{t.bannerText}</span>
+              <span className="ds-c-usa-banner__header-action" aria-hidden="true">
+                {t.bannerActionText}
+              </span>
+              <button
+                onClick={this.handleToggleBanner}
+                className="ds-c-usa-banner__button"
+                aria-expanded={this.state.isBannerOpen}
+                aria-controls="gov-banner"
+              >
+                <span className="ds-c-usa-banner__button-text">{t.bannerActionText}</span>
+              </button>
+            </p>
+          </div>
         </header>
         <div className="ds-c-usa-banner__content" id="gov-banner" hidden={!this.state.isBannerOpen}>
           <div className="ds-u-display--flex ds-u-flex-direction--column ds-u-sm-flex-direction--row ds-u-flex-wrap--nowrap">
```

Comment:
Is it easier to use a grid here? I know this is the existing code but I'm guessing it might be more succinct

Reply:
That's an idea, will check it out

Reply:
Have tidied the scss. I'm making another branch to refactor the banner to grid-layout

Reply:
@nichia why do you need another branch to do this? wont it be a simple replacement of some classes?

Reply:
I was thinking we would want to keep the markup as close to if not exactly like the USWDS Banner so we can reference their styles and get any future updates in an easier way?